src/js/Controllers/BCController.js

```diff
@@ -98,6 +102,16 @@ class BCController {
             case "GetSystemTime":
                 this.listener.send(RpcFactory.GetSystemTime(rpc.id))
                 return null
+
+            case "OnAppCapabilityUpdated":
+                if (rpc.params.appCapability.appCapabilityType === 'VIDEO_STREAMING') {
+                    var vsc = rpc.params.appCapability.videoStreamingCapability;
+                    if (!vsc.additionalVideoStreamingCapabilities) {
```

Comment:
also check if vsc is valid

Reply:
done in https://github.com/smartdevicelink/generic_hmi/pull/345/commits/81b4a86651d856a228ef2179a99da9dfc5b269d6